include/drivers/can.h

```diff
@@ -98,7 +98,7 @@ enum can_mode {
 	/*Controller is in loopback mode (receive own messages)*/
 	CAN_LOOPBACK_MODE,
 	/*Combination of loopback and silent*/
-	CAN_SILENT_LOOPBACK_MODE,
```

Comment:
It's generally nice to leave the comma on the last element, as it removes gratuitous diffs if new elements are added below.